assets/swaggerui/swagger.json

```diff
@@ -302,120 +338,19 @@
           }
         }
       }
-    },
-    "/v1/users/{id}/password": {
-      "patch": {
-        "description": "If user's old passowrd is correct, it will be replaced with new password.",
-        "tags": [
-          "users"
-        ],
-        "summary": "Changes user's password.",
-        "operationId": "pwChange",
-        "parameters": [
-          {
-            "type": "integer",
-            "description": "id of user",
-            "name": "id",
-            "in": "path",
-            "required": true
-          },
-          {
-            "description": "Request body",
-            "name": "request",
-            "in": "body",
-            "required": true,
-            "schema": {
-              "$ref": "#/definitions/pwChange"
-            }
-          }
-        ],
-        "responses": {
-          "200": {
-            "$ref": "#/responses/ok"
-          },
-          "400": {
-            "$ref": "#/responses/errMsg"
-          },
-          "401": {
-            "$ref": "#/responses/err"
-          },
-          "403": {
-            "$ref": "#/responses/err"
-          },
-          "500": {
-            "$ref": "#/responses/err"
-          }
-        }
-      }
     }
   },
   "definitions": {
     "AccessRole": {
       "description": "AccessRole represents access role type",
       "type": "integer",
-      "format": "int8",
-      "x-go-package": "github.com/ribice/gorsk/internal"
+      "format": "int64",
+      "x-go-package": "github.com/ribice/gorsk/pkg/utl/model"
     },
-    "Credentials": {
-      "description": "Credentials contains login request",
+    "HTTP": {
```

Comment:
maybe need a `swagger:ignore` in code
`pkg/api/password/transport` and `pkg/api/user/transport`